scss/bootstrap-grid.scss

```diff
@@ -16,6 +16,9 @@ html {
   box-sizing: inherit;
 }
 
+// Deprecate
+@import "mixins/deprecate";
```

Comment:
Can you move this down to the other mixins?

Reply:
Sure, I had it there initially but then I went to check where you guys have put it on the [nixon's file](https://github.com/twbs/bootstrap/blob/a716fb03f965dc0846df479e14388b1b4b93d7ce/scss/_mixins.scss#L9) and so it at the top. Should I move it down there too?

Reply:
The new spot for it is right, thanks! Just wanted it at the top of the mixins section :).